packages/sources/coingecko/src/endpoint/crypto.ts

```diff
@@ -101,20 +139,38 @@ const handleBatchedRequest = (
 }
 
 export const execute: ExecuteWithConfig<Config> = async (request, context, config) => {
-  const validator = new Validator(request, inputParameters, {}, { overrides })
-
-  const endpoint = validator.validated.data.endpoint
+  const validator = new Validator(
+    request,
+    inputParameters,
+    {},
+    {
+      overrides,
+      symbolToIdOverrides,
+    },
+  )
+  const endpoint = validator.validated.data.endpoint ?? DEFAULT_ENDPOINT
```

Comment:
This should be already filled from the `Selector`. Are you seeing scenarios where it isn't?

Reply:
I am not quite sure what you mean here.  The only change from the original code on line 104 was to add symbolToIdOverrides that are specified in the adapter's symbolToIdOverrides.json file